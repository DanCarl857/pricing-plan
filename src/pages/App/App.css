.plan-body {
  font-family: 'Lato', sans-serif;
  text-align: center;
}

.custom-card {
  padding: 30px;
  border-radius: 10px;
  -webkit-box-shadow: 10px 10px 24px -4px rgba(0,0,0,0.75);
  -moz-box-shadow: 10px 10px 24px -4px rgba(0,0,0,0.75);
  box-shadow: 10px 10px 24px -4px rgba(0,0,0,0.75);
  width: 350px;
  height: 400px;
  margin-top: 25px;
}

.custom-card:hover {
  border: 3px solid #4885ed;
  transform: scaleY(1);
  cursor: pointer;
}

.custom-card--big {
  background-color: #4885ed;
  padding: 30px;
  border-radius: 10px;
  -webkit-box-shadow: 10px 10px 24px -4px rgba(0,0,0,0.75);
  -moz-box-shadow: 10px 10px 24px -4px rgba(0,0,0,0.75);
  box-shadow: 10px 10px 24px -4px rgba(0,0,0,0.75);
  height: 600px;
  margin-top: 25px;
  color: #fff;
}

.custom-card--big:hover {
  color: #4885ed;
  background-color: #fff;
  cursor: pointer;
}
.custom-card__price {
  font-size: 35px;
  font-weight: 400;
}

.custom-card__header-bronze {
  color: #E9D758;
}

.custom-card__header-gold {
  color: #d3abb0;
}

.custom-card__header-silver {
  color: #4e5283;
}

.header__text {
  margin: 50px 0px;
}

.header__sub-text {
  width: '60%'
}

.pricing {
  margin-top: 50px;
}

.info {
  margin-top: 70px;
}

.button {
  position: absolute;
  left: 0;
  bottom: 0;
}

.offer {
  color: #4885ed;
  font-weight: 600;
}