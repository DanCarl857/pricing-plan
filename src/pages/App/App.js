import React from 'react'
import './App.css'
import { Container, Row, Col, Button } from 'react-bootstrap'

function App() {
  return (
    <div className="plan-body container">
      <div className="header">
        <h2 className="header__text">Pricing Plans</h2>
        <h6 className="header__sub-text">Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat</h6>
      </div>
      <hr/>

      <section className="pricing">
        <div className="row m-auto text-center">
          <div className="custom-card col-md-4 col-xs-12">
            <h5 className="custom-card__header-bronze">Bronze</h5>
            <p className="custom-card__price">10.000 FCFA</p>
            <p className="custom-card__body">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
            <ul class="list-unstyled mb-5 position-relative">
              <li><b>10</b> users included</li>
              <li><b>2 GB</b> of storage</li>
              <li><b>No </b>Email support</li>
              <li><b>No Help center access</b></li>
            </ul>
            <Button variant="secondary" block className="button" size="lg">Get Started</Button>
          </div>

          <div className="custom-card--big col-md-4 col-xs-12">
            <h5 className="custom-card__header-gold">Gold</h5>
            <p className="custom-card__price">35.000 FCFA</p>
            <p className="custom-card__body">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
            <ul class="list-unstyled mb-5 position-relative">
              <li><b>1000</b> users included</li>
              <li><b>20 GB</b> of storage</li>
              <li><b>24/7 </b>Email support</li>
              <li><b>Help center access</b></li>
            </ul>
            <Button variant="primary" block className="button" size="lg">Get Started</Button>
          </div>

          <div className="custom-card col-md-4 col-xs-12">
            <h5 className="custom-card__header-silver">Silver</h5>
            <p className="custom-card__price">20.000 FCFA</p>
            <p className="custom-card__body">Lorem ipsum dolor sit amet, consectetur adipiscing elit</p>
            <ul className="list-unstyled position-relative mb-30">
              <li><b>100</b> users included</li>
              <li><b>10 GB</b> of storage</li>
              <li><b>6hours </b>Email support</li>
              <li><b>Limited Help center access</b></li>
            </ul>
            <Button variant="secondary" block className="button" size="lg">Get Started</Button>
          </div>
        </div>
      </section>

      <Container className="info">
        <div>
          <p>Limited offer. Save 20% of Gold plans. Get coupon <span className="offer">here.
            </span></p>
        </div>
      </Container>
    </div>
  );
}

export default App;
