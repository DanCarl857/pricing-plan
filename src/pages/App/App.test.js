import React from 'react'
import App from './App'
import renderer from 'react-test-renderer'

describe(`UI: Render correctly`, () => {
  it('renders correctly', () => {  
    const tree = renderer  
    .create(<App />)  
    .toJSON();  
      expect(tree).toMatchSnapshot();  
    });
})
