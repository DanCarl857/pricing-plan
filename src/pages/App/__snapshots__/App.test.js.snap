// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UI: Render correctly renders correctly 1`] = `
<div
  className="plan-body container"
>
  <div
    className="header"
  >
    <h2
      className="header__text"
    >
      Pricing Plans
    </h2>
    <h6
      className="header__sub-text"
    >
      Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat
    </h6>
  </div>
  <hr />
  <section
    className="pricing"
  >
    <div
      className="row m-auto text-center"
    >
      <div
        className="custom-card col-md-4 col-xs-12"
      >
        <h5
          className="custom-card__header-bronze"
        >
          Bronze
        </h5>
        <p
          className="custom-card__price"
        >
          10.000 FCFA
        </p>
        <p
          className="custom-card__body"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit
        </p>
        <ul
          class="list-unstyled mb-5 position-relative"
        >
          <li>
            <b>
              10
            </b>
             users included
          </li>
          <li>
            <b>
              2 GB
            </b>
             of storage
          </li>
          <li>
            <b>
              No 
            </b>
            Email support
          </li>
          <li>
            <b>
              No Help center access
            </b>
          </li>
        </ul>
        <button
          className="button bronze btn btn-secondary btn-block btn-lg"
          disabled={false}
          type="button"
        >
          Get Started
        </button>
      </div>
      <div
        className="custom-card--big col-md-4 col-xs-12"
      >
        <h5
          className="custom-card__header-gold"
        >
          Gold
        </h5>
        <p
          className="custom-card__price"
        >
          35.000 FCFA
        </p>
        <p
          className="custom-card__body"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit
        </p>
        <ul
          class="list-unstyled mb-5 position-relative"
        >
          <li>
            <b>
              1000
            </b>
             users included
          </li>
          <li>
            <b>
              20 GB
            </b>
             of storage
          </li>
          <li>
            <b>
              24/7 
            </b>
            Email support
          </li>
          <li>
            <b>
              Help center access
            </b>
          </li>
        </ul>
        <button
          className="button btn btn-primary btn-block btn-lg"
          disabled={false}
          type="button"
        >
          Get Started
        </button>
      </div>
      <div
        className="custom-card col-md-4 col-xs-12"
      >
        <h5
          className="custom-card__header-silver"
        >
          Silver
        </h5>
        <p
          className="custom-card__price"
        >
          20.000 FCFA
        </p>
        <p
          className="custom-card__body"
        >
          Lorem ipsum dolor sit amet, consectetur adipiscing elit
        </p>
        <ul
          className="list-unstyled position-relative mb-30"
        >
          <li>
            <b>
              100
            </b>
             users included
          </li>
          <li>
            <b>
              10 GB
            </b>
             of storage
          </li>
          <li>
            <b>
              6hours 
            </b>
            Email support
          </li>
          <li>
            <b>
              Limited Help center access
            </b>
          </li>
        </ul>
        <button
          className="button btn btn-secondary btn-block btn-lg"
          disabled={false}
          type="button"
        >
          Get Started
        </button>
      </div>
    </div>
  </section>
  <div
    className="info container"
  >
    <div>
      <p>
        Limited offer. Save 20% of Gold plans. Get coupon 
        <span
          className="offer"
        >
          here.
        </span>
      </p>
    </div>
  </div>
</div>
`;
